src/olympia/amo/utils.py

```diff
@@ -490,7 +490,7 @@ def normalize_string(value, strip_punctuation=False):
 def slug_validator(
     s,
     ok=SLUG_OK,
-    lower=True,
+    lower=False,
```

Comment:
That function follows `slugify()` closely, only adding arguments that matter for validator purposes. `slugify()` defaults to `lower=True` so I don't think we should change that.

Reply:
I did it because we always passed `lower=False` everywhere we used `slug_validator` but if we want those redundant kwargs I can change it back?

Reply:
I was tempted to drop all the kwargs apart from `message` so we're directly relying on slugify defaults.

Reply:
If you mean drop all kwargs except `message` and always pass `lower=False` to `slugify` then yes, that sounds ok.